docs/en_US/install/cemu.md

```diff
@@ -10,10 +10,12 @@
 
 Cemu 2.0 has official built-in support for Pretendo as of October 10, 2022. Head to the Cemu GitHub [releases](https://github.com/cemu-project/Cemu/releases) page and download the latest Cemu experimental release (tagged as `Pre-release`). Only `Cemu 2.0-5 (Experimental)` and above is supported at the moment. Additionally you may build Cemu from source using the provided [build instructions](https://github.com/cemu-project/Cemu/blob/main/BUILD.md)
 
+After you can follow [Cemu's online guide](https://cemu.cfw.guide/online-play.html) to dump the online files from your Wii U after you logged in to your Pretendo Account on it.
+
 ## Connecting to Pretendo
 Once you have Cemu setup for online play navigate to `Options > General settings > Account`. You should now see a section titled `Network Service`. Select your PNID from the `Active account` menu and select the `Pretendo` Network Service option. Cemu should now be connected to Pretendo's servers
 
 <center><img src="/assets/images/docs/install/cemu/network-services-settings.png"/></center>
 
 ## Miiverse
-Cemu has limited to no Miiverse support as of now. Currently no Miiverse features, including the Miiverse applet as well as Miiverse features in games, will function.
\ No newline at end of file
+Cemu has limited to no Miiverse support as of now. Currently no Miiverse features, including the Miiverse applet as well as Miiverse features in games, will function.
```

Comment:
This is no longer true, some Miiverse stuff does work and you *can* get Cemu to load the applet if you modify it

```suggestion
Cemu has limited to no Miiverse support as of now. Some in game features may work, but this is not guaranteed. The Miiverse applet does not work in official builds.
```